class-omnisend-lifterlmsaddon.php

```diff
@@ -0,0 +1,150 @@
+<?php
+/**
+ * Plugin Name: Omnisend for LifterLMS Add-On
+ * Description: A LifterLMS add-on to sync contacts with Omnisend. In collaboration with Omnisend for WooCommerce plugin it enables better customer tracking
```

Comment:
So is Omnisend for woocommere is required or only omnisend core?

Reply:
only Omnisend core is required. Changed both plugin descriptions.